.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,12 @@
+The issue tracker isn't the place for general support. For questions and
+technical assistance, come ask the
+[Bokeh mailing list](https://groups.google.com/a/continuum.io/forum/#!forum/bokeh).
+
+For defects or deficiencies, please provide:
+* Description of expected behavior and the observed behavior
+* self-contained example code that generates the issue (if possible)
+* stack traceback and/or browser console error output
+* python, bokeh and browser versions
```

Comment:
we maybe need to reference the print versions thing we have: https://github.com/bokeh/bokeh/blob/84423bac8a9897236e8e84f22492bd77760f10ea/bokeh/util/testing.py#L23